### PR TITLE
changes for pcm-656

### PR DESCRIPTION
**Pull request by pbathia**

### CODE REVIEW

**Comment by pbathia** [comment]
@renujhamtani @sshumake Please review and merge.

**Comment by renujhamtani** [comment]
Looks good!!

**Comment by sshumake** [comment]
Looks good, this will need to be manually merged.

**Comment by renujhamtani** [comment]
Looks good! Merging!

File: app/chromed.html [hunk 1]
```diff
@@ -221,6 +221,7 @@
     window.deps.push("/ascension/controllers/caseEscalations.js");
     window.deps.push("/ascension/controllers/caseSearch.js");
     window.deps.push("/ascension/controllers/searchSolutions.js");
+    window.deps.push("/ascension/controllers/caseAssociates");
```
Comment (renujhamtani) [inline app/chromed.html]: @pbathia This shoud be caseAssociates.js ...the main apge is not loading because of this.

**Pull request merged**